docs/source/install_linux.rst

```diff
@@ -51,11 +51,10 @@ Step 1
 ~~~~~~
 
 This assumes you have a Homebrew installation of GCC installed already 
-(we are using GCC 6.3.0). These instructions have been tested on OSX 10.11. MacOS 10.12 
-will not build CMake with GCC anymore, so these instructions won't work 
```

Comment:
Thanks to @rafaelmudafort for pointing this line out. I'm not sure about this. Hence I'm taking this out.